client/my-sites/customer-home/cards/education/mastering-gutenberg/index.jsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { Button, Dialog } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import DismissibleCard from 'blocks/dismissible-card';
```

Comment:
We can probably drop the DismissableCard option, since I think we'll end up favoring letting folks cycle through the educational cards, right @danhauk ?

Reply:
Switched from a `DismissibleCard ` to an ordinary `Card` 👍